src/renderers/shared/shared/__tests__/ReactUpdates-test.js

```diff
@@ -13,9 +13,9 @@
 
 var React;
 var ReactDOM;
-var ReactDOMFeatureFlags;
 var ReactTestUtils;
 var ReactUpdates;
+var ReactDOMFeatureFlags = require('ReactDOMFeatureFlags');
```

Comment:
Uh... I don't remember. I thought I did this in a file where the branch wrapped a `describe` block but that's not true of this file.